src/kadmin/ktutil/ktutil.c

```diff
@@ -149,6 +150,7 @@ void ktutil_add_entry(argc, argv)
         }
         if ((strlen(argv[i]) == 2) && !strncmp(argv[i], "-k", 2)) {
             kvno = (krb5_kvno) atoi(argv[++i]);
+            use_kvno++;
```

Comment:
Hi, that patch is originally from here: http://krbdev.mit.edu/rt/Ticket/Display.html?id=3349